### PR TITLE
Add macOS 13 Ventura and remove Catalina support

### PR DESCRIPTION
**Pull request by athomas**

### CODE REVIEW

**Comment by parlough** [comment]
Thanks for updating this, looks good to me. Should this be immediate or wait for the next release?

File: src/get-dart/index.md [hunk 1]
```diff
@@ -75,10 +75,10 @@ The Dart SDK is supported on Windows, Linux, and macOS.
 ### macOS
 
 * **Supported versions:** Latest three major versions.
-  As of November 2021, the following versions are supported:
-  - macOS 10.15 (Catalina)
+  As of October 2022, the following versions are supported:
```
Comment (atsansone) [inline src/get-dart/index.md:78]: **issue:** Use active voice.
```suggestion
Dart supports the following macOS versions as of October 2022:
```

**Review by atsansone — APPROVED** [review]
LGTM with the following change included.

**Comment by athomas** [comment]
I haven't found any issues on Ventura so far.

**Review by mit-mit — APPROVED** [review]

**Pull request merged**